.gitignore

```diff
@@ -93,6 +93,7 @@ HistomicsTKDocker.code-workspace
 MANIFEST.in
 .pre-commit-config.yaml
 run_tests_code_cleanup.sh
+seliniumTest.py
```

Comment:
Where did this come from? Are you using selenium in some manner?

Reply:
Yeah, I wrote some code to automate the UI processes after a new docker is deployed. The code can go until a cli job is submitted with/without an ROI. 

I got the spelling wrong for the filename.